### PR TITLE
Add v1.1.0 of mattermost-plugin-nps to the Marketplace

### PR DESCRIPTION
**Pull request by hanzei**
#### Summary
Screenshot from 2020-09-18 21-29-50

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-28520

### CODE REVIEW

File: plugins.json [hunk 1]
```diff
@@ -1832,6 +1832,50 @@
...
+    "download_url": "https://plugins-store.test.mattermost.com/release/mattermost-plugin-nps-v1.1.0.tar.gz",
+    "release_notes_url": "",
```
Comment (larkox) [inline plugins.json:1839]: Missing this URL. I suppose it is not important.

**Review by larkox — APPROVED** [review]
LGTM, just a small thing, which I am not sure how important it is.

File: plugins.json (hunk 1 above)
Comment (hanzei) [reply plugins.json:1839]: Given that we are bundling NPS with 5.28 it doesn't seem important. I will still add it for the next release.

**Review by DHaussermann — APPROVED** [review]
Tested and passes
- Tested upgrade path
- Tested fresh install path
- Version number correctly shows as `1.1.0`
- Icon looks fine in all themes

LGTM!

**Pull request merged**